ref/general/hardening-intro.adoc

```diff
@@ -63,7 +63,7 @@ To learn more about network vulnerabilities and ways to prevent network intrusio
 * *Use a firewall*: Use a firewall to shield production Open Liberty servers so that they don't run in the demilitarized zone (DMZ).
 The DMZ is an unsecured network segment that exists between internal and external networks.
 Firewalls help prevent security intrusions by monitoring network traffic and blocking specific traffic based on security rules.
-* *Enable the Transport Security feature*: The link:/docs/ref/feature/#transportSecurity.html[Transport Security feature] enables the link:/docs/ref/feature/#ssl.html[SSL feature] to support TLS connections and adds the ability to configure outbound transports.
+* *Enable the Transport Level Security feature*: The link:/docs/ref/feature/#transportSecurity.html[Transport Level Security feature] enables TLS for HTTP connections and adds the ability to configure outbound transports with TLS.
```

Comment:
Because the feature is officially called Transport Security in the OL docs, I think that's how we should refer to it: https://openliberty.io/docs/ref/feature/#transportSecurity-1.0.html.

I did change the end of the sentence to what you have ("enables TLS for HTTP connections and adds the ability to configure outbound transports with TLS.)